editors/vim/Portfile

```diff
@@ -6,25 +6,13 @@ PortGroup           github 1.0
 # The vim port should always follow the patch level used in the MacVim port. This
 # is because they both share the same set of configuration files and ensures
 # that the user's configuration files will always work with both.
-set vim_version     9.0
+set vim_version    9.1
+set vim_patchlevel 0000
+set port_revision  0
 
-if {${os.platform} eq "darwin" && ${os.major} <= 14} {
-    # Fallback to previous port version, for 10.10 and earlier
-    # https://trac.macports.org/ticket/67209
-    set vim_patchlevel 0472
-    set port_revision  1
-
-    checksums       rmd160  ee809eb300deaf735c86b6e5b7f4bc4a8fe79a41 \
-                    sha256  7185577d6cd3708b62b2079b69a29215bb6a6048070d34e960d2d565c18b8a9b \
-                    size    16851486
-} else {
-    set vim_patchlevel 1897
-    set port_revision  0
-
-    checksums       rmd160  03c9bc4d54789cbec843fbfbfa342de2f6af4384 \
-                    sha256  8b86cfb2b609e7a7e386b88a89b01522354990d4eef27e7001cc607d49d62ebf \
-                    size    17268335
-}
+checksums           rmd160  aed3a40e2fb0a96fc22c554b2fd102fb3369d560 \
+                    sha256  f0cdc73963122d052a666d88f787ec5b295f75ebdc3dfa5aa5a48f8afa92d98f \
+                    size    17604158
 
 github.setup        vim vim ${vim_version}.${vim_patchlevel} v
 revision            ${port_revision}
```

Comment:
```suggestion
revision            0
```

Reply:
I've removed the `port_revision` variable in 3809d06b514ed9c36b87457d4e0b62940381f47c.

I also moved the `checksums` section later to be consistent with the layout of the MacVim port.